src/litserve/loops.py

```diff
@@ -495,9 +495,6 @@ def __init__(self):
         self._context = {}
 
     def get_batch_requests(self, lit_api: LitAPI, request_queue: Queue, max_batch_size: int, batch_timeout: float):
-        if max_batch_size <= 1:
```

Comment:
so that continuous batching can work like single continuous loop without setting max_batch_size. the underlying functions works with batch size [1, INF].